app/models/service_orchestration.rb

```diff
@@ -27,14 +27,14 @@ def orchestration_stack_status
   end
 
   def deploy_orchestration_stack
-    deploy_stack_options = stack_options
     job_options = {
-      :create_options            => deploy_stack_options,
+      :create_options            => stack_options,
```

Comment:
👍 Since the [stack_options](https://github.com/ManageIQ/manageiq/blob/master/app/models/mixins/service_orchestration_options_mixin.rb#L3-L5) method does caching there is no need to setup the `deploy_stack_options` variable.